lib/spack/spack/stage.py

```diff
@@ -578,6 +585,9 @@ def expand_archive(self):
                          '{stage}\n\tdestination : {destination}'.format(
                              stage=source_path, destination=destination_path
                          ))
+                # TODO: why not symlink here? i.e.
+                #   os.symlink(source_path, destination_path)
+                # If move is intentional the comments above have to be changed
                 shutil.move(source_path, destination_path)
```

Comment:
This comment is unrelated to this pull request so I will remove it. However the reason I added this comment in the first place what that the current comments (which refer to symlinks) are outdated/confusing.

@alalazo you introduced `shutil.move` instead of `os.symlink` in 093b8317998c. Can you remember why moving is necessary here? As the stages are presumably on the same file system symlinks could work. If `shutil.move` is necessary I think the comments should be updated → I can do this in a separate PR.

Reply:
You can update this to `symlink`.

That being said ping @alalazo: in case you have any objection.

Reply:
@kljohann Sorry for the delayed reply, lost this notification among many others (and @scheibelp thanks for the ping). I didn't introduce this, I just lost it in a merge conflict, and then put it back. 

More precisely: I had the merge conflict in #298 (see [comment](https://github.com/LLNL/spack/pull/298#issuecomment-174884034)) and the change was introduced in #293 by @trws in 4ae98f8b214ea491a50aa2ddf3d995625d1b35f7. His commit message seems to suggest it's better to leave it as it is :smile: